proposals/0167-app-services.md

```diff
@@ -978,44 +903,41 @@ The HMI will be responsible for the actual selection and activation of app servi
 
 ```xml
 <function name="GetAppServiceRecords" messagetype="request">
-  <param name="serviceTypes" type="Common.ServiceType" mandatory="false">
-    <description>If included, only service records of supplied type will be returned in response. If not included, all service records for all types will be returned.</description>
+  <param name="serviceType" type="String" mandatory="false">
+    <description>The type of service that is to be offered by this app. See AppServiceType for known enum equivalent types. Parameter is a string to allow for new service types to be used by apps on older versions of SDL Core.</description>
   </param>
 </function>
 
 <function name="GetAppServiceRecords" messagetype="response">
-  <param name="serviceRecords" type="Common.AppServiceRecord" array="true"mandatory="false">
+  <param name="serviceRecords" type="Common.AppServiceRecord" array="true" mandatory="false">
     <description>All app service records of requested type.</description>
   </param>
 </function>
 
 
 <function name="AppServiceActivation" messagetype="request">
-  <param name="serviceId" type="String" mandatory="true" >
+  <param name="serviceID" type="String" mandatory="true" >
     <description>The ID of the service that should have an activation event take place on</description>
   </param>
-  <param name="serviceType" type="Common.ServiceType" mandatory="true">
-    <description>The service type the service should have the activation event occur on</description>
+  <param name="serviceType" type="String" mandatory="true">
+    <description>The type of service that is to be offered by this app. See AppServiceType for known enum equivalent types. Parameter is a string to allow for new service types to be used by apps on older versions of SDL Core.</description>
```

Comment:
Should be removed, as was done in the response.

Reply:
Updated